qiskit/extensions/quantum_initializer/ucrz.py

```diff
@@ -87,7 +87,17 @@ def ucz(self, angle_list, q_controls, q_target):
     if num_contr != len(q_controls):
         raise QiskitError("Number of controlled rotations does not correspond to the number"
                           " of control-qubits.")
-    return self.append(UCZ(angle_list), [q_target] + q_controls, [])
+    return self.append(UCRZGate(angle_list), [q_target] + q_controls, [])
+
+
+def ucz(self, angle_list, q_controls, q_target):
+    """
+    Deprecated version of ucrz.
+    """
+    import warnings
+    warnings.warn('qc.ucz is deprecated, use qc.ucrz instead!', DeprecationWarning, 2)
```

Comment:
This message does not match the style of other deprecation messages.
```suggestion
    warnings.warn('The QuantumCircuit. ucz() method is deprecated as of 0.14.0, and '
                  'will be removed no earlier than 3 months after that release date. '
                  'You should use the QuantumCircuit. ucrz() method instead.',
                  DeprecationWarning, stacklevel=2)
```